### PR TITLE
Tweaked lavaland cursed katana

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
tweak: The lavaland katana is now more obviously "cursed" in its name.
tweak: If you manage to drop the cursed katana, you will no longer be
torn apart by mind demons. It no longer disappears on drop, leaving the
option for someone else to pick it up.
/:cl:

Saw a miner come in, got the katana removed, but the brain trauma was
permament for the rest of the shift. Should be slightly more explicit in
its cursed nature if you look before picking it up. Leaving it behind
when dropped allows for some other sucker to pick it up.

### CODE REVIEW

**Comment by Fikou** [comment]
i agree with the dropdel removal, but this will break on wizard cursed event which is the other way of obtaining the katana, making a "cursed cursed katana" and also i think you should keep the curse even after you drop it, you lose the downside if you can just spend 1 minute in the medbay to get rid of it

**Comment by D4C-420** [comment]
> i agree with the dropdel removal, but this will break on wizard cursed event which is the other way of obtaining the katana, making a "cursed cursed katana" and also i think you should keep the curse even after you drop it, there is no downside if you can just spend 1 minute in the medbay to get rid of it, it was a deliberate choice 😳

Let the trauma be removable with lobectomy and stay even after dropping the katana. The curse thing will probably catch up to the miner (or whatever sod took the chainsword ) by then and get in a few hits. A slow doctor might just end up killing them if they don't keep pace.

**Comment by ShibaInuLord** [comment]
> 
> 
> > i agree with the dropdel removal, but this will break on wizard cursed event which is the other way of obtaining the katana, making a "cursed cursed katana" and also i think you should keep the curse even after you drop it, there is no downside if you can just spend 1 minute in the medbay to get rid of it, it was a deliberate choice 😳
> 
> Let the trauma be removable with lobectomy and stay even after dropping the katana. The curse thing will probably catch up to the miner (or whatever sod took the chainsword ) by then and get in a few hits. A slow doctor might just end up killing them if they don't keep pace.


if you didnt know, the curse is extremely slow, like very very slow, the time I had the katana I only saw it like twice in around 40 minutes

**Comment by nianjiilical** [comment]
this might be too much of a nerf (buff?) but would it be possible to make the katana only curse itself to you if you actually hit something with it

losing your arm and being cursed for misclicking on an item that appears in usually-beneficial chests sucks a little

**Comment by Pugie** [comment]
Cursed cursed katana sounds funny though

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -253,18 +253,22 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	return(BRUTELOSS)
 
 /obj/item/katana/cursed
-	slot_flags = null
-	item_flags = DROPDEL
+	name = "cursed katana"
+	desc = "Power comes at a price."
+	slot_flags = null  // cannot escape NODROP by putting on belt.
+	var/datum/brain_trauma/magic/stalker/curse
```
Comment (Jared-Fogle) [inline code/game/objects/items/weaponry.dm:259]: If my understand is correct, this is going to hard delete if the trauma is removed by something other than dropping the cursed katana (in this case, admin heals, as well as the staff of healing).

You should instead either hold this as a weakref or make the cursed katana stateless, and just run `cure_trauma_type` in `dropped`.

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Pull request closed without merging**